ffi/blitbuffer.lua

```diff
@@ -596,8 +596,10 @@ local MASK_INVERSE = 0x02
 local SHIFT_INVERSE = 1
 local MASK_ROTATED = 0x0C
 local SHIFT_ROTATED = 2
-local MASK_TYPE = 0xF0
+local MASK_TYPE = 0x70
```

Comment:
This'll need a matching change in blitbuffer.c ;).

Reply:
Yep. Provided I'll get ad-hoc rotation going on correctly (ie no SW rotations ever), this is one of the places where C blitter can benefit a lot (the operation can be parallelized on neon for batches of 12 pixels).